africanlii/urls.py

```diff
@@ -68,6 +68,10 @@
     ),
     path("feeds/all.xml", CoreDocumentAtomSiteNewsFeed(), name="atom_feed"),
     path("i18n/", include("django.conf.urls.i18n")),
+    re_path(
+        r"^(?P<expression_frbr_uri>akn/.*)/$",
```

Comment:
Using re_path is fine, maybe tweak the regex closer to our frbr_uri format

Reply:
Does the normal django `path` segment not work? That allows basically anything, including slashes.

For reversing, remember that you'll need to strip the leading `/` from the FRBR URI.

Reply:
Also, just call this `frbr_uri`. We're going to need to add suspport for processing it as both an FRBR expression URI, and an FRBR work URI.

Reply:
Changed. @ro6ley I think that the regex is sufficient for this purporse since it captures everything that starts with akn/ and strips off the trailing slash. It will simply throw a 404 if no document exists with that frbr_uri.